### PR TITLE
VACMS-0000: Updates Drupal to 9.3.8.

### PR DESCRIPTION
**Pull request by ndouglas**
https://www.drupal.org/project/drupal/releases/9.3.8

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -25187,5 +25187,5 @@
     "platform-overrides": {
         "php": "7.4"
     },
-    "plugin-api-version": "2.2.0"
+    "plugin-api-version": "2.0.0"
```
Comment (tjheffner) [inline composer.lock:25190]: just verifying that this bit is intentional... everything else seems to move up a version save for this one

Comment (ndouglas) [reply composer.lock:25190]: I think Dependabot tends to bump this up to 2.2.0 and most of us use 2.0.0.  It's not a separate package and has no effect  I can discern.

**Review by cweagans — DISMISSED** [review]

**Review by tjheffner — DISMISSED** [review]

**Comment by ndouglas** [comment]
Thanks ❤️

**Comment by ndouglas** [comment]
Merging because it was already approved, just GH fail.

**Pull request merged**